martypy/Marty.py

```diff
@@ -703,6 +703,22 @@ def get_add_on_status(self, add_on_name_or_id: Union[int, str]) -> Dict:
         '''
         return self.client.get_add_on_status(add_on_name_or_id)
 
+    def add_on_query(self, addOnName: str, dataToWrite: bytes, numBytesToRead: int) -> Dict:
```

Comment:
```suggestion
    def add_on_query(self, add_on_name: str, data_to_write: bytes, num_bytes_to_read: int) -> Dict:
```
Almost no other methods use camel case so I think it's better to stay consistent.